src/feed/filters.py

```diff
@@ -50,6 +53,48 @@ def filter_queryset(self, request, queryset, view):
         # Fall back to default ordering behavior for any explicit ordering parameters
         return super().filter_queryset(request, queryset, view)
     
+    def _apply_include_ended_filter(self, request, queryset, view):
+        """
+        Apply include_ended filtering to exclude OPEN items past their end_date.
+        
+        Args:
+            request: The request object
+            queryset: The queryset to filter
+            view: The view instance
+            
+        Returns:
+            Filtered queryset
+        """
+        # Check include_ended parameter - only apply if not on "Previously Funded" tab
+        fundraise_status = request.query_params.get('fundraise_status', '').upper()
+        is_previously_funded_tab = fundraise_status == 'CLOSED'
+        
+        include_ended = request.query_params.get('include_ended', 'true').lower() == 'true'
+        # Don't apply include_ended filter on Previously Funded tab
+        if is_previously_funded_tab:
+            include_ended = True
+        
+        # Apply include_ended filtering
+        if not include_ended:
+            # Check view type to determine which model to filter
+            if getattr(view, 'is_grant_view', False):
+                model_class = Grant
+                open_status = Grant.OPEN
+            else:
```

Comment:
Important!
Do we need to be concerned with feed items that are NOT grants or funding?